src/rpc/net.cpp

```diff
@@ -122,7 +122,9 @@ static RPCHelpMan getpeerinfo()
     return RPCHelpMan{
         "getpeerinfo",
         "Returns data about each connected network peer as a json array of objects.",
-        {},
+        {
+            {"peer_id", RPCArg::Type::NUM, RPCArg::Optional::OMITTED, "Only return info for the specified peer."}
```

Comment:
Since we have to iterate peers to find the right one, maybe this should be a list (unordered_set in C++)